### DISCUSSION TITLE
Connecting to a XGA monitor

### DISCUSSION THREAD

**Discussion by cslater1972** [community/community#127391] [Programming Help]
### Select Topic Area

Question

### Body

Hi All I have recently purchased a ZX Spectrum Pico kit with a Raspberry Pi 2040 chip with a VGA output I can get it to work on a small VGA monitor, however i wanted it to work on my TV which has a PC port. on further examination I have have found out that the TV PC port is XGA and not VGA it there any options available to support the kit to use an XGA monitor. 
I am really new to this and would really appreciate any pointers and support.
the last time I built a ZX Spectrum was in 1983 (showing my age lol) and that was a 16k kit form

**Comment by LiteBrite82** [comment]
Hi @cslater1972 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this product. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
